src/decisionengine_modules/GCE/transforms/GceFigureOfMerit.py

```diff
@@ -41,8 +41,15 @@ def transform(self, data_block):
             it = row["InstanceType"]
             entry_name = row["EntryName"]
 
-            occupancy_df = gce_occupancy[((gce_occupancy.AvailabilityZone == az) & (gce_occupancy.InstanceType == it))]
-            occupancy = float(occupancy_df["Occupancy"].values[0]) if not occupancy_df.empty else 0
+            try:
+                occupancy_df = gce_occupancy[
+                    ((gce_occupancy.AvailabilityZone == az) & (gce_occupancy.InstanceType == it))
+                ]
+            except:
+                occupancy = 0
+                self.logger.debug(f"GceFigureOfMerit transform: Looks like no VMs runnig in GCE.")
```

Comment:
If you don't include any variable there is no need for `f""`, just the string is OK `""`.

Reply:
This is also taken care of. 
(I just forgot to erase f earlier)